src/modules/commander/commander_params.c

```diff
@@ -671,31 +671,53 @@ PARAM_DEFINE_INT32(COM_ARM_MIS_REQ, 0);
 PARAM_DEFINE_INT32(COM_POSCTL_NAVL, 0);
 
 /**
- * Arm authorization parameters, this uint32_t will be split between starting from the LSB:
- * - 8bits to authorizer system id
- * - 16bits to authentication method parameter, this will be used to store a timeout for the first 2 methods but can be used to another parameter for other new authentication methods.
- * - 7bits to authentication method
- * 		- one arm = 0
- * 		- two step arm = 1
- * * the MSB bit is not used to avoid problems in the conversion between int and uint
+ * Require arm authorization to arm
+ *
+ * The default allows to arm the vehicle without a arm authorization.
  *
- * Default value: (10 << 0 | 1000 << 8 | 0 << 24) = 256010
- * - authorizer system id = 10
- * - authentication method parameter = 1000 msec of timeout
- * - authentication method = during arm
  * @group Commander
+ * @unit s
+ * @min 0
  */
-PARAM_DEFINE_INT32(COM_ARM_AUTH, 256010);
```

Comment:
We might want to consider migrating this parameter for anyone actually using it.

@bkueng have you seen any actual arm authorization usage?

Reply:
> Looks fine to me, although it would be good if anyone else actually using arm authorization could take a closer look. Note that none of the automated testing (or manual for that matter...) touches arm authorization, so please test it thoroughly. Contributing testing for this functionality would also be quite valuable.

It is well tested before rebase (on v1.10) in our ecosystem (without parameter refactoring).
Tested these commits after rebase on master but not that much. Only the first method have been tested. I will test the second one too.

For testing contribution, I will have a look on how to do that but will probably not have time to do that for now.

Reply:
I can have a look on migrating if you consider it necessary

Reply:
Yes, migration would be beneficial.

Reply:
I will add that

Reply:
done